Adding a wrap-up post for GSoC 2015.

febuiles opened the pull request:
This should cover the successful projects for this year's GSOC.

Feel free to make any changes/corrections!

febuiles committed:
Adding a wrap-up post for GSoC 2015.

rafaelfranca commented on _posts/2015-09-24-gsoc-2015-wrapping-up.markdown:
```diff
@@ -0,0 +1,72 @@
+Maps](https://developer.mozilla.org/en-US/docs/Tools/Debugger/How_to/Use_a_source_map) to
+Sprockets 4. With Source Maps users can see a readable version of their code inside Developer
+Tools (Javascript, CSS, etc.) after it's been minified or compiled. Check out Andrei's
+[sprockets](https://github.com/Andreis13/sprockets-rails/commits/master) and
```
Almost all work from Andrei is already merged and we can point to them directly. Let me get the links

https://github.com/rails/sprockets/commits?author=Andreis13
https://github.com/rails/sprockets-rails/commits?author=Andreis13
https://github.com/rails/sass-rails/commits?author=Andreis13

febuiles committed:
Updating links for the source maps project.

febuiles committed:
Updating post date.

febuiles commented:
I've updated the PR with @rafaelfranca's suggestions and I modified the post date. Please let me know if there's anything else we need to do. (cc. @chancancode).

chancancode commented:
:heart:
